lib/booktype/skeleton/base_settings.py.original

```diff
@@ -197,6 +197,8 @@ MIDDLEWARE_CLASSES = (
     'django.middleware.csrf.CsrfViewMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
+    'booktype.api.middleware.AuthMiddleware',
```

Comment:
It looks good to me in general. Although there is just one small detail, `booktype.api.middleware.AuthMiddleware` won't work if you don't add `booktype.api.auth.ApiBackend` as backend authentication. Also I think we shouldn't enable this feature by default, it should be just enabled for the clients that need it. Will fix this myself and merge it.